Formula/chronograf.rb

```diff
@@ -0,0 +1,102 @@
+require "language/node"
+
+class Chronograf < Formula
+  desc "Open source monitoring and visualization UI for the TICK stack"
+  homepage "https://www.influxdata.com/time-series-platform/chronograf/"
+  url "https://github.com/influxdata/chronograf.git",
+      :tag => "1.1.0-beta2",
```

Comment:
@bfontaine unfortunately it looks like the "stable" version of Chronograf probably won't be released until January. I'm happy to close this PR and reopen once that has been released. Any feedback on how to fix the npm error in the tests or other general feedback would be welcome.